### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by anupkgurung**
home

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,189 @@
...
+    <link rel="stylesheet" href="./style/commons.css">
+    <link rel="stylesheet" href="./style/header.css">
+    <link rel="stylesheet" href="./style/homepage.css">
+    <title>Home</title>
+</head>
+<body>
+    <div>
+        <div id="" class="header-bar">
+            <div class="topbar">
+                <div><img class="logo" src="./image/logo.jpg" alt=""></div>
```
Comment (Naveenchandar) [inline index.html:20]: Please provide alt description for accessibility.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+                        <li class="m-1 list-style"><a>History & Humanities</a></li>
+                        <li class="m-1 list-style"><a>Self Help & Personal Development</a></li>
+                        <li class="m-1 list-style"><a>Society & social sciences</a></li>
+                        <li class="m-1 list-style"><a>Sports</a></li>
+                        <li class="m-1 list-style"><a>Technology & Engineering</a></li>
+                        <li class="m-1 list-style"><a>Travel & holiday</a></li>
+                    </ul>
+                </aside>               
+            </div>
+            <div id="content" class="">
```
Comment (Naveenchandar) [inline index.html:80]: If class attribute is not used, you can remove it.

File: style/header.css [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+    background-color: var(--white);
+    top: 0;
+    padding: 0.5rem 3rem;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;    
+}
+.icon-badge {
+    margin-left: 2.7rem;
+    margin-top: -2.4rem;
```
Comment (Naveenchandar) [inline style/header.css]: Please use margin shorthand

File: style/header.css [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+    padding: 0rem 25rem;
+}
+.navbar-list > li {
+    padding: 1rem;
+}
+.nav {
+    padding: 0rem;
+}
+.nav-box-shadow {
+    box-shadow: -1px 5px 5px -4px black;
```
Comment (Naveenchandar) [inline style/header.css:40]: Please use hex code for consistency and if color is repeated, you can declare a variable and utilise it.

File: index.html (hunk 1 above)
Comment (anupkgurung) [reply index.html:20]: sure !!

File: index.html (hunk 2 above)
Comment (anupkgurung) [reply index.html:80]: i had to use it but forgot to add the class name thanks.

File: style/header.css (hunk 3 above)
Comment (anupkgurung) [reply style/header.css]: yeah would change that.

**Review by laxmikanthkonthum — APPROVED** [review]
Looks good to me!

**Review by maitrakhatri — APPROVED** [review]
looks good to me

**Pull request merged**